### PR TITLE
feat: prepare action comments with links

### PR DESCRIPTION
**Pull request by benjaminParisel**
For pr-comments-with-links action, we extract the code in a real action to ease the maintenance.

With this PR, we are as isofunctional as we have actually (with at least some unit test)

We update the comment, the new design is:
image

Cover https://github.com/bonitasoft/bonita-documentation-site/issues/766

### CODE REVIEW

File: packages/doc-contribs/pr-comments-with-links/src/CommentsWithLinks.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+> The following pages were updated, please ensure that the display is correct:
+${links?.updated.map((item) => `> ${item}`).join("\n")}`;
+    }
+
+    let deletedSection = "";
+    if (links?.deleted.length > 0) {
+      deletedSection = `
+###  :mag: Check redirects
+> [!warning]
+> At least one page has been renamed, moved or deleted in the Pull Request. Make sure to add [**aliases**](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#use-alias-to-create-redirects) and **verify that the following links redirect to the right location**:
```
Comment (tbouffard) [inline packages/doc-contribs/pr-comments-with-links/src/CommentsWithLinks.ts:91]: **suggestion:** by Deepl 😸 
```suggestion
> At least one page has been renamed, moved or deleted in the Pull Request. Make sure you add [**aliases**](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#use-alias-to-create-redirects) and **check that the following links redirect to the right place**:
```

File: packages/doc-contribs/pr-comments-with-links/src/CommentsWithLinks.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   * const links = {
+   *   updated: ['- [ ] [page1](http://example.com/component/1.0/page1)'],
+   *   deleted: ['- [ ] [page2](http://example.com/component/1.0/page2)']
+   * };
+   * ```
+   */
+  buildComment(links: Links) {
+    const header = "## :memo: Contribution Summary \n\n";
+    const preface =
+      "In order to merge this pull request, you need to check your updates with the following url.\n\n";
```
Comment (tbouffard) [inline packages/doc-contribs/pr-comments-with-links/src/CommentsWithLinks.ts:76]: **suggestion:** by Deepl 💯 
```suggestion
      "To merge this Pull Request, you need to check your updates with the following URL.\n\n";
```

File: packages/doc-contribs/pr-comments-with-links/src/main.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      version: version,
+    });
+    links.deleted = commentsWithLinks.prepareLinks({
+      files: deletedFiles,
+      siteUrl: siteUrl,
+      component: componentName,
+      version: version,
+    });
+    if (links.deleted.length === 0 && links.updated.length === 0) {
+      core.info(`⚠️ No page will be updated or deleted`);
```
Comment (tbouffard) [inline packages/doc-contribs/pr-comments-with-links/src/main.ts:56]: **suggestion:** by Deepl
```suggestion
      core.info(`⚠️ No updated or deleted pages were detected`);
```

Comment (benjaminParisel) [reply packages/doc-contribs/pr-comments-with-links/src/main.ts]: :heavy_check_mark: Done in last commit

**Review by tbouffard — APPROVED** [review]
LGTM 🥇

**Pull request merged**